### ISSUE TITLE
NTR, pulmo: Necrotizing pulmonary granulomatosis

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6385]
**Preferred term label:**
Necrotizing pulmonary granulomatosis

**Synonyms**
caseating pulmonary granulomatosis

**Definition (free text, please give PubMed ID)**
A granuloma that is associated with necrotic changes. Caseation necrosis is defined as a region in granulomas with eosinophilic, granular and cheese-like cellular debris with necrosis. The word caseous itself means "pertaining or related to cheese", and comes from the Latin word caseus, meaning cheese. Comment: In HE staining, it is characterized by acellular pink areas of necrosis surrounded by a granulomatous inflammatory process. 

**Parent term (use hpo.jax.org/app)**
Pulmonary granulomatosis HP:0030250

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Necrotizing sarcoid granulomatosis (PMID: 26040776), Tuberculosis (TB; ORPHA:3389), fungal infections (Hot tube lung)

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
added term
